src/Workspaces/Core/Portable/Utilities/AsyncLazy`1.cs

```diff
@@ -294,6 +294,13 @@ private Request CreateNewRequest_NoLock()
 
         public override Task<T> GetValueAsync(CancellationToken cancellationToken)
```

Comment:
I tend to agree with that. If the value's already there, the caller probably *doesn't* want to cancel.